src/app/components/restaurants-table/restaurants-table.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, OnInit } from '@angular/core';
+import { MatTableDataSource } from '@angular/material/table';
+import { MatDialog } from '@angular/material/dialog';
+import { RestaurantService } from '../../services/restaurant.service';
+import { IRestaurant } from '../../models/restaurant.model';
+import { AddRestaurantModalComponent } from './add-restaurant-modal/add-restaurant-modal.component';
+import { EditRestaurantModalComponent } from './edit-restaurant-modal/edit-restaurant-modal.component';
+
+@Component({
+  selector: 'app-restaurants-table',
+  templateUrl: './restaurants-table.component.html',
+  styleUrl: './restaurants-table.component.scss',
+})
+export class RestaurantsTableComponent implements OnInit {
+  public displayedColumns: string[] = [
```

Comment:
If you have raw data, and a database, use the database to store your data instead
If you feel it's simple data and don't want to store it in a database and make an api call for it, create another file that will hold this simple data for you to use